routes/posts.js

```diff
@@ -0,0 +1,21 @@
+import { Router } from "express";
+import { User } from "../model/User.js";
+
+import { auth } from "./verifyToken.js"
+
+const router = Router();
+const verify =  auth;
```

Comment:
you may import this as ( auth as verify). Do this only if there maybe a conflict with other modules/ functions